llvm/utils/TableGen/OptionParserEmitter.cpp

```diff
@@ -572,5 +572,5 @@ static void EmitOptParser(const RecordKeeper &Records, raw_ostream &OS) {
   OS << "\n";
 }
 
-static TableGen::Emitter::Opt X("gen-opt-parser-defs", EmitOptParser,
+static TableGen::Emitter::Opt X("gen-opt-parser-defs", EmitOptionParser,
```

Comment:
Not renaming the option yet as that can cause lot of changes potentially. I could look into allowing register an alias of the option in the TG backend, but not sure if it's worth it.